python-checks/src/main/java/org/sonar/python/checks/IgnoredParameterCheck.java

```diff
@@ -59,6 +60,7 @@ public void initialize(Context context) {
           .filter(assignment -> assignment.symbol.usages().stream().filter(Usage::isBindingUsage).count() > 1)
           // no usages in unreachable blocks
           .filter(assignment -> !isSymbolUsedInUnreachableBlocks(lva.getLiveVariables(block), unreachableBlocks, assignment.symbol))
+          .filter((assignment -> !isUsedInSubFunction(assignment.symbol, functionDef)))
```

Comment:
I'm not sure why we change that rule in a PR which is supposed to be about rule S3827.

Reply:
This PR modified some code in common between LVA and DVA (defined variable analysis).
More specifically, it's because in UsageVisitor we don't visit anymore FunctionDefinitions

https://github.com/SonarSource/sonar-python/blob/a76e2295744d0cc56d257408bef3b4116cce28ee/python-frontend/src/main/java/org/sonar/python/cfg/fixpoint/UsageVisitor.java#L43